pkg/service/service.go

```diff
@@ -0,0 +1,133 @@
+package service
```

Comment:
not a single test for this whole package :-(

Reply:
tests already added in `occlient_test.go`